cmds/exp/tftpd/main.go

```diff
@@ -0,0 +1,225 @@
+// Copyright 2012-2024 the u-root Authors. All rights reserved
```

Comment:
Really nitty, feel free to ignore
```suggestion
// Copyright 2012-2025 the u-root Authors. All rights reserved
```

Reply:
fxed